### PR TITLE
Update distributed-tracing.mdx with caveat on 128 bit IDs

### PR DESCRIPTION
**Pull request by lkysow**

### CODE REVIEW

File: website/content/docs/connect/distributed-tracing.mdx [hunk 1]
```diff
@@ -238,3 +238,26 @@ config to take effect.
    Envoy not yet having support.
 
 1. Tracing is only supported with Envoy proxies, not the built-in proxy.
+
+1. When configuring the Zipkin tracer in `envoy_tracing_json`, set `"trace_id_128bit": true` if your application is configured to generate 128-bit trace IDs. For example:
```
Comment (blake) [inline website/content/docs/connect/distributed-tracing.mdx:242]:
```suggestion
1. When configuring the Zipkin tracer in `envoy_tracing_json`, set [`trace_id_128bit`](https://www.envoyproxy.io/docs/envoy/v1.21.0/api-v3/config/trace/v3/zipkin.proto#envoy-v3-api-field-config-trace-v3-zipkinconfig-trace-id-128bit) to  `true` if your application is configured to generate 128-bit trace IDs. For example:
```
What do you think about linking to the Envoy docs for this parameter?

File: website/content/docs/connect/distributed-tracing.mdx [hunk 2]
```diff
@@ -238,3 +238,26 @@ config to take effect.
...
+           "@type": "type.googleapis.com/envoy.config.trace.v3.ZipkinConfig",
+           "collector_cluster": "zipkin",
+           "collector_endpoint_version": "HTTP_JSON",
+           "collector_endpoint": "/api/v2/spans",
+           "shared_span_context": false,
+           "trace_id_128bit": true
+         }
+       }
+     }
+   }
```
Comment (blake) [inline website/content/docs/connect/distributed-tracing.mdx:260]:
```suggestion
{
  "http": {
    "name": "envoy.tracers.zipkin",
    "typedConfig": {
      "@type": "type.googleapis.com/envoy.config.trace.v3.ZipkinConfig",
      "collector_cluster": "zipkin",
      "collector_endpoint_version": "HTTP_JSON",
      "collector_endpoint": "/api/v2/spans",
      "shared_span_context": false,
      "trace_id_128bit": true
    }
  }
}
```

**Review by blake — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/566985.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 4df488b1d3dc348006596465727f907c7b756a3e onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 4df488b1d3dc348006596465727f907c7b756a3e onto `release/1.11.x` succeeded!

**Pull request merged**